dist/js/app.js

```diff
@@ -255,49 +255,16 @@ function setDownloadSection() {
 function onNightlyLoad() {
   /* eslint-enable no-unused-vars */
 
-
   populateNightly(); // run the function to populate the table on the Nightly page.
 
-  // logic for the realtime search box:
-  var search = document.getElementById("search");
-  var searchError = document.getElementById("search-error");
-
-  search.onkeyup = (function() {
-    var filter, found, table, tr, td, i, j;
-    filter = search.value.toUpperCase();
-    table = document.getElementById("nightly-table");
-    tr = table.getElementsByTagName("tr");
-    for (i = 0; i < tr.length; i++) {
-        td = tr[i].getElementsByTagName("td");
-        for (j = 0; j < td.length; j++) {
-            if (td[j].innerHTML.toUpperCase().indexOf(filter) > -1) {
-                found = true;
-            }
-        }
-        if (found) {
-            tr[i].style.display = "";
-            found = false;
-        } else {
-          if (tr[i].id != 'table-header'){
-            tr[i].style.display = "none";
-          }
-        }
-    }
-    if(document.getElementById('table-parent').offsetHeight < 45) {
-      table.style.visibility = "hidden";
-      searchError.className = "";
-    } else {
-      table.style.visibility = "";
-      searchError.className = "hide";
-    }
-  });
-
 }
 
 
 // NIGHTLY PAGE FUNCTIONS
 
 function populateNightly() {
+  const tableHead = document.getElementById("table-head");
```

Comment:
Could this be raised as an issue and resolved in a later PR?

I agree that it should be split, but it isn't a new problem being introduced by this PR, so I'd prefer to get this merged and split the function after.